hazelcast/src/main/java/com/hazelcast/map/impl/MapService.java

```diff
@@ -19,24 +19,13 @@
 import com.hazelcast.core.DistributedObject;
```

Comment:
No, It is not provided for all operations, just for map/transactional map for now.